Source/carthage/CopyFrameworks.swift

```diff
@@ -3,15 +3,29 @@ import Commandant
 import Foundation
 import Result
 import ReactiveSwift
+import Curry
 
 /// Type that encapsulates the configuration and evaluation of the `copy-frameworks` subcommand.
 public struct CopyFrameworksCommand: CommandProtocol {
+    public struct Options: OptionsProtocol {
+        public let automatic: Bool
+        public let useFrameworkSearchPaths: Bool
+        public let isVerbose: Bool
+        
+        public static func evaluate(_ mode: CommandMode) -> Result<Options, CommandantError<CarthageError>> {
+            return curry(self.init)
+                <*> mode <| Option(key: "auto", defaultValue: false, usage: "infers and copies linked frameworks automatically")
```

Comment:
add *experimental* notice here.

I'm suggesting this in case people start depending on some behavior we have finalized yet.

Reply:
Added `(experimental)` note.